examples/source/1.components/amp-script/index.html

```diff
@@ -10,158 +22,225 @@
-    if (isValid) {
-      submitButton.removeAttribute('disabled');
-    } else {
-      submitButton.setAttribute('disabled', '');
-    }
-  });
-</script>
-
-  <!--
-      ## Detecting Android vs iOS in AMP
-
-      `amp-script` is great to implement functionality not offered by AMP out-of-the-box. Here is a sample that checks if the current device is using Android or iOS.
-  -->
-  <amp-script layout="container" script="user-agent-script" class="amp-script-sample">
-    <div>Your user-agent is: <span id="user-agent">other</span></div>
-  </amp-script>
-
-  <!-- The script implementation retrieves the mobile operating system from the user agent string. -->
-  <script id="user-agent-script" type="text/plain" target="amp-script">
-    function getMobileOperatingSystem() {
-        const userAgent = navigator.userAgent;
-        if (/android/i.test(userAgent)) {
-          return "Android";
+        const button = document.getElementById('hello-inline');
+
+        button.addEventListener('click', () => {
+          const h1 = document.createElement('h1');
+          h1.textContent = 'Hello World!';
+          document.body.appendChild(h1);
+        });
+      </script>
+    </div>
+    <!-- [tip type="note"]
+    `amp-script` passes its children to your script as the virtual DOM - not the entire DOM. To your script, those children *are* the DOM.
+    Thus, `document.body` refers to what's inside the `amp-script` tag, not the actual `body`. `document.body.appendChild(...)` actually adds an element inside the `amp-script` element.
+    [/tip]
+    -->
+    <!--
+        ## Using the fetch API
+
+        `amp-script` supports the fetch API.
+        `amp-script` allows us to update the page on load if it knows that the script can't change the component's height. Here, we use `fixed-height` layout, and we specify the `height` in an HTML attribute. See [the documentation](/content/amp-dev/documentation/components/reference/amp-script-v0.1.md#script-hash) for details.
+    -->
+    <div>
+      <amp-script layout="fixed-height" height="36" script="time-script" class="sample">
+        <div>The time at page load was: <span id="time" class="answer-text"></span></div>
+      </amp-script>
+
+      <script id="time-script" type="text/plain" target="amp-script">
+        const fetchCurrentTime = async () => {
+          const response = await fetch('<% hosts.platform %>/documentation/examples/api/time');
+          const data = await response.json();
+          const span = document.getElementById('time');
+          span.textContent = data.time;
         }
-        // iOS detection from: http://stackoverflow.com/a/9039885/177710
-        if (/iPad|iPhone|iPod/.test(userAgent) && !window.MSStream) {
-          return "iOS";
+
+        fetchCurrentTime();
+      </script>
+    </div>
+    <!-- 
+        ## Showing live data
+
+        You can also use `setInterval()` or `setTimeout` to get fresh data.
```

Comment:
I'd love to have an websockets sample as well

Reply:
That was on my list of samples to create. I'm hesitating because there's already an example on the documentaiton page. But we can totally do it. If so, perhaps that can be used for a live-blogging or stock price example.

Reply:
Messaging would be a great example!